src/backend/XmlHttpRequest.ts

```diff
@@ -69,11 +75,12 @@ export default class XmlHttpRequest extends BaseFileSystem implements FileSystem
     if (typeof(opts.index) === "string") {
       XmlHttpRequest.FromURL(opts.index, cb, opts.baseUrl, false);
     } else {
-      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false));
+      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false, opts.use));
```

Comment:
constructor no longer throws an exception (since I am using a boolean instead of a string), so... wait, no, it still can throw an exception if `listingUrlOrObj` is a string! I guess that was a pre-existing "bug". I think this means we'll need to catch and handle construction errors in both `.Create` and in `.FromURL`.